hack/config.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer"
+docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirtd"
```

Comment:
It looks like travis will support caching soon: https://github.com/travis-ci/travis-ci/issues/5358#issuecomment-252741180 - that could help.